docs/faq.md

```diff
@@ -177,3 +177,32 @@ There is no golden rule, but you can follow these recommendations:
 
 This warning only occurs if you use non-default settings in your vite config that can cause problems in combination with prebundleSvelteLibraries.
 You should not use prebundleSvelteLibraries during build or for ssr, disable one of the incompatible options to make that warning (and subsequent errors) go away.
+
+<!-- the following header generates an anchor that is used in logging, do not modify!-->
+
+### deprecated "svelte" field
+
+In the past, Svelte recommended using the custom "svelte" field in package.json to allow libraries to point at .svelte source files.
+This field requires a custom implementation to resolve, so you have to use a bundler plugin and this plugin needs to implement resolving.
+Since then, node has added support for [conditional exports](https://nodejs.org/api/packages.html#conditional-exports), which have more generic support in bundlers and node itself. So to increase the compatibility with the wider ecosystem and reduce the implementation needs for current and future bundler plugins, it is recommended that packages use the "svelte" exports condition.
+
+Example:
+
+```diff
+// package.json
+- "svelte": "src/index.js"
++ "exports": {
++   "./package.json": "./package.json",
```

Comment:
Maybe we should not recommend export `package.json` too as it's not necessarily needed.

Reply:
this is what rollup plugin svelte did recommend and before we moved it to vitefu vite-plugin-svelte still used that export to load package.json if possible. does vitefu not do it and always read from fs?

Reply:
yeah vitefu will always read from the fs. I don't think there's perf issues in practice though. There's https://github.com/sveltejs/rollup-plugin-svelte/pull/198 in rollup plugin svelte to stop warning if `package.json` is not exported too.

Reply:
we should definitely have the plugin read from the fs. `rollup-plugin-svelte`'s behavior is incredibly annoying. that being said, I would leave this as is for now. `rollup-plugin-svelte` still will issue a warning if it's not exported, so users should probably export it for the time being until we update `rollup-plugin-svelte`

Reply:
removed the package.json line and also kept the svelte field as neutral instead of remove. We should not encourage them to remove it for now as that could break older tooling that still relies on it.